### ISSUE TITLE
gRPC enable

### ISSUE THREAD

**Issue by Alinda3** [cloudflare/terraform-provider-cloudflare#1103]
### Current Terraform and Cloudflare provider version

It would be great if we could enable gRPC through terraform in cloudflare. 

(https://support.cloudflare.com/hc/en-us/articles/360050483011-Understanding-Cloudflare-gRPC-support)

### Description

A resource "grpc setting" to enable/disable this settings

### Use cases

.

### Potential Terraform configuration

```hcl
...
```
.

**Comment by jacobbednarz** [comment]
grpc doesn't currently have public API support so we'll need to wait for that before we can add support for it.

**Comment by erikschul** [comment]
Is there no progress on this?

**Comment by finkinfridom** [comment]
still no news?

**Comment by jacobbednarz** [comment]
This issue has been closed as we are now tracking this internally with service teams directly. If you would like an update or to be notified when/if the product ships with this change, please reach out to Cloudflare Support or your account team who can watch the internal feature request for you.

**Comment by ssarioglu** [comment]
still waiting

**Comment by james-callahan** [comment]
This still seems to be missing?
FWIW in the dashboard toggling gRPC sends a request to `/api/v4/zones/$zone_id/flags/products/protocols/changes` which appears to be an undocumented endpoint.

**Comment by rp-fintopio-acc** [comment]
still waiting

**Comment by jhutchings1** [comment]
Reopening per customer conversation, to discuss at triage with the team today.

**Comment by ricardoduval** [comment]
Well, at least the discussion has been reopened. We also have this requirement, to enable gRPC. And our whole Cloudflare configuration is done with Terraform. It's a shame we have to resource to a manual step for something so trivial.

**Comment by AndreMiras** [comment]
> This issue hasn't been updated in a while. If it's still reproducing, please comment to let us know. Thank you!

I believe it's still an issue indeed, API isn't exposed as far as I know

**Comment by chadxz** [comment]
ran into this today :(

**Comment by callmesangio** [comment]
> This issue hasn't been updated in a while. If it's still reproducing, please comment to let us know. Thank you!

Yes, please, keep this open until gRPC management is supported by API and TF provider.

**Comment by dgski** [comment]
Just another humble voice begging for this feature 🥺

**Comment by SC4RECOIN** [comment]
would be nice to have this plz

**Comment by nrlkirby** [comment]
This would help us avoid an otherwise manual and easily forgotten step, would love to have terraform support

**Comment by xmarcelet-saas** [comment]
also showing my interest for this
